autoload/lsp.vim

```diff
@@ -484,6 +484,9 @@ function! lsp#default_get_supported_capabilities(server_info) abort
     " Sorted alphabetically
     return {
     \   'textDocument': {
+    \       'callHierarchy': {
+    \           'dynamicRegistration': v:true,
+    \       },
```

Comment:
Sorry. The vim-lsp does not support dynamicRegistration yet. So it should be turned to `v:false`.